core/lib/tracehouse/trace-processor.js

```diff
@@ -489,20 +489,22 @@ class TraceProcessor {
    */
   static findMainFramePidTids(mainFrameInfo, keyEvents) {
     const frameCommittedEvts = keyEvents.filter(evt =>
```

Comment:
nit: this no longer has just frame committed events, maybe rename to something like `frameProcessEvts`?